src/components/DevfileEditor/index.tsx

```diff
@@ -14,8 +14,8 @@ import React from 'react';
 import { connect, ConnectedProps } from 'react-redux';
 import { AppState } from '../../store';
 import { DisposableCollection } from '../../services/helpers/disposable';
-import { ProtocolToMonacoConverter, MonacoToProtocolConverter } from 'monaco-languageclient/lib/monaco-converter';
-import { languages, IEditorConstructionOptions } from 'monaco-editor-core/esm/vs/editor/editor.main';
+import { ProtocolToMonacoConverter, MonacoToProtocolConverter } from './monaco-converter';
```

Comment:
I don't see how changes in this component are related to moving the project to yarn2. Could you point out why they are needed?

Reply:
It depends how yarn 2 resolve packages. Old import was broken. I did not find a good solution(except 'nodeLinker: node-modules' - is not a good idea as for me) for fix it and decide that the best one will be just copy the target file add a link.

@akurinnoy  If you have some better idea, please, provide a PR with changes.

Reply:
When my local build was failing on this file, adding this resolution to the package.json fixed it for me:

```json
  "resolutions": {
    ...
    "vscode-languageserver-protocol": "3.15.3"
  }
```

you may have a different error though.

Reply:
Thanks all. I have fixed it.